pkgs/development/python-modules/setoptconf/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "setoptconf";
+  version = "0.2.0";
```

Comment:
this package hasn't had an update in almost 6 years.... I don't think it belongs in a rolling release package set

Reply:
where would you place it? it's needed by prospector

Reply:
just have it alongside prospector in pkgs/development/tools/prospector/setoptconf.nix.

and just do:

```
  setoptconf = python3.pkgs.callPackage ./setoptconf.nix
```